dashboard/app/models/script.rb

```diff
@@ -1598,6 +1599,10 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
     summary
   end
 
+  def unit_without_lesson_plans
```

Comment:
nit: in ruby, I think it's a convention to append a `?` to methods that return a boolean. Can you change this to `unit_without_lesson_plans?`